README.md

```diff
@@ -18,10 +18,11 @@ The current velocity of the player can be displayed with 2 main elements:
 
 ## Manual checkpoints
 
-A total of **3 manual checkpoints** can be set by the player, by default:
-- `F2`, `F3` and `F4` will set the first, second and third checkpoint;
-- `2`, `3` and `4` will load the first, second and third checkpoint;
-- `Z` will reset the 3 checkpoints.
+Unlimited checkpoints can be set by the player, by default:
+- `1` will create a checkpoint;
+- `2` will load the checkpoint;
+- `3` will remove the current checkpoint, allowing you to go back to the previous checkpoint;
+- `Z` will clear all checkpoints.
```

Comment:
True, I forgot about AZERTY, maybe something like "R" would be better?

I think the reason it is Y in the code is because originally I had a debug event to happen when I pressed Y and I guess I forgot to change it, oops...